src/main/java/darwin/gui/Main.java

```diff
@@ -0,0 +1,29 @@
+package darwin.gui;
+
+import darwin.Darwin;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+public class Main extends Application {
+
+    @Override
+    public void start(Stage stage) {
+        try {
+            stage.setMinHeight(220);
+            stage.setMinWidth(417);
+            FXMLLoader fxmlLoader = new FXMLLoader(Main.class.getResource("/view/MainWindow.fxml"));
+            AnchorPane ap = fxmlLoader.load();
+            Scene scene = new Scene(ap);
+            stage.setScene(scene);
+            fxmlLoader.<MainWindow>getController().setDarwin(new Darwin("./tasks.txt"));
+            stage.show();
```

Comment:
Would be good to space out your code based on their logical parts, instead of one huge chunk of code.